src/common/components/signUpModal/signUpModal.tpl.html

```diff
@@ -1,53 +1,126 @@
 <div class="col-md-12 col-xs-12">
-  <form>
+  <div class="alert alert-danger" role="alert" ng-messages="$ctrl.signUpErrors" ng-if="$ctrl.hasError">
+    <div ng-message="400" translate>An error occurred during sign up.<br />Please Check your Email Address and Password and try again.</div>
+    <div ng-message="403" translate>An error occurred during sign up.<br />This is most likely caused by the use of a reserved domain name in your email address. You might try to
```

Comment:
What does a `reserved domain name in your email address` mean? I feel like if I'm not sure what it means, users will be confused by the error...